misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-grssignum/src/main/java/de/metas/camel/externalsystems/grssignum/to_grs/api/model/JsonBPartner.java

```diff
@@ -119,6 +122,7 @@ public JsonBPartner(
 			@JsonProperty("METASFRESHURL") final @Nullable String metasfreshURL,
 			@JsonProperty("KREDITORENNR") final @Nullable Integer creditorId,
 			@JsonProperty("DEBITORRENNR") final @Nullable Integer debtorId,
```

Comment:
here we still have my typo that i noticed the other day and informed you about in mattermost..jsut be sure that in case of a merge-conflict you go with `DEBITORENNR` (just two `R`s)

Reply:
yep, we addressed it in https://github.com/pevesoft/metasfresh/issues/174.